src/Inspector/Controller/OpcacheController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Debug\Api\Inspector\Controller;
+
+use Psr\Http\Message\ResponseInterface;
+use Yiisoft\DataResponse\DataResponseFactoryInterface;
+
+use function Safe\opcache_get_status;
```

Comment:
Is this a new dependency?

Reply:
I guess, it was meant to be just `opcache_get_status` from root namespace. Right?